server/common/src/test/java/org/eclipse/lsp/cobol/common/mapping/ExtendedDocumentTest.java

```diff
@@ -52,6 +52,7 @@ void init() {
     copybookUri = UUID.randomUUID().toString();
     document = new ExtendedDocument(TEXT, documentUri);
     copybook = new ExtendedText(COPYBOOK, copybookUri);
+    document2 = new ExtendedDocument(copybook, documentUri);
```

Comment:
is this required? doesn't seems to be used

Reply:
removed